deploy/managed-upgrade-operator-config/10-managed-upgrade-operator-configmap.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: openshift-managed-upgrade-operator
 data:
   config.yaml: |
+    ocmBaseUrl: "${{OCM_BASE_URL}}"
```

Comment:
`OCM_BASE_URL` needs to be defined in the template parameters.
https://github.com/openshift/managed-cluster-config/blob/b3837d88798a0c20715d057ed1efccceb3e3484e/scripts/templates/template.yaml#L3